airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -119,27 +122,52 @@ public AirbyteCatalog discover(JsonNode config) throws Exception {
     }
   }
 
+  protected void updateSchemaFocus(JsonNode config) {
+    if (config.has("schemaFocus")) {
+      this.schemaFocus = config.get("schemaFocus").asBoolean();
```

Comment:
usually in java we don't put the `this` in front of fields. the exception is in constructors where you often have naming collisions between the parameter passed into the constructor and the field itself. so in this case `this.dialect = dialect;` we use it it but in the cases on 127 and 129 you usually would not include the `this`. line 80 can go either way. i usually include it because it makes line up with the rest of the field assignments, but i'm not sure what the best practice is in that case.